### PR TITLE
Thor/reskin 1

### PR DESCRIPTION
**Pull request by melissahie**
- [x] simplified CTA to just Explore API
- [x] removed some unnecessary elements
- [x] removed illustration
- [x] removed partner sign up call out since we are no longer accepting partners
- [x] add topbar for QBC announcement
- [ ] **NOTE:** Still need to remove "TradeGecko" from the actual API doc

 --

### BEFORE
image

### AFTER
image
image

--

### API Docs

**NOTE:** Still need to remove "TradeGecko" from the actual API doc, PR got too long already

image

### CODE REVIEW

**Review by franciak — APPROVED** [review]
Will the headline still saying `TradeGecko` be on the separate PR too? (Start building with TradeGecko API...)

**Comment by melissahie** [comment]
@franciak how did i miss that -_-

File: source/includes/home_cta.erb [hunk 1]
```diff
@@ -1,14 +1,11 @@
 <div class="home-cta-container">
   <div class="home-cta-wrapper">
     <h1>
-      Build Apps for over 200,000 merchants or just for your own team
+      Build Apps for over 200,000 merchants, or just for your own team
     </h1>
     <div class="home-cta_button-wrapper">
-      <a href="https://go.tradegecko.com/register" target="blank_" class="btn btn-big btn-primary" >
-          Sign Up Now
```
Comment (abdullacontractor) [inline source/includes/home_cta.erb:8]: Not entirely sure what happening here the reskin for this look quite extensive. But why have we removed a sign up button?

**Review by abdullacontractor — APPROVED** [review]
Wow this looks incredible!

File: source/includes/home_cta.erb (hunk 1 above)
Comment (melissahie) [reply source/includes/home_cta.erb:8]: it didn't make sense to begin with, anyone who comes here should just have 1 goal in mind: to explore API Docs.

**Review by alexsmee — APPROVED** [review]

**Pull request merged**